core/translations/en.json

```diff
@@ -299,13 +299,13 @@
     "ethereum__amount_sent": "Amount sent:",
     "ethereum__contract": "Contract:",
     "ethereum__data_size_template": "Size: {0} bytes",
-    "ethereum__gas_limit": "Gas limit:",
```

Comment:
Removing the `:` and adding them dynamically when constructing layouts works but brings undesirable changes to `fr` language because french uses space-colon.

We can either ignore it or duplicate the strings to have e.g.
`ethereum__gas_limit` and `ethereum__gas_limit_colon`.

Reply:
Yes, I was thinking about duplication for this reason, but then I worried it would add more to the flash size and this was a decent compromise. But maybe I am wrong, maybe only strings that are actually used make it to the image, then it is not an issue.

Speaking of this, I just read about it now, it seems the space before semicolon in French should be a non-breaking space (makes sense!). Now that we support that, I should also replace all occurrences with the non-breaking space!

Reply:
An even better way is introducing something like `label__with_semicolon_template = "{0} :"` and using that everywhere. I'll do that.